### DISCUSSION TITLE
Suppressing specific errors with config.yaml is not working

### DISCUSSION THREAD

**Discussion by grezp** [clangd/clangd#1055] [Q&A]
Hello,

I'm trying to suppress errors using a config.yaml as the clangd documentation suggest. However, the changes I make are not taking affect.

I've created a simple config:

```
Diagnostics:
  Suppress: '*'
```

I expected this to suppress all diagnostics. However, this does nothing.

Is there something wrong with my config? Any suggestions?

**Comment by HighCommander4** [comment]
Could you share [clangd logs](https://clangd.llvm.org/troubleshooting#gathering-logs), and the location of your config file please?

**Reply by grezp** [reply]
I'm not sure what happened. After I closed and opened my shell all errors were suppressed.

Separate question:
[How can I disable these errors?](https://imgur.com/a/3ELDmFe)
Or can you point me to where a list of diagnostic codes exist. I couldn't find it in the docs.

I assumed it would work as a glob, but it seems that I need a diagnostic code.

**Reply by HighCommander4** [reply]
If you turn on clangd's verbose logs with `--log=verbose`, the LSP diagostic notifications should include the diagnostic codes for each diagnostic. Feel free to post a log if you're having trouble finding it.

Not sure where there is a list, except in the [source](https://searchfox.org/llvm/rev/5ac257da3559f6bd716330da512735ac07364802/clang/include/clang/Basic/DiagnosticLexKinds.td#191).
